ios/demo/src/MapViewController.m

```diff
@@ -32,22 +32,40 @@ - (void)mapView:(TGMapViewController *)mapView didLoadSceneAsync:(NSString *)sce
     [mapView setPosition:newYork];
 }
 
-- (void)mapView:(TGMapViewController *)mapView didSelectFeature:(NSDictionary *)feature atScreenPosition:(CGPoint)position
+- (void)mapView:(TGMapViewController *)mapView didSelectLabel:(TGLabelPickResult *)labelPickResult atScreenPosition:(CGPoint)position
 {
+    if (!labelPickResult) { return; }
 
-    // Not feature selected
-    if (feature.count == 0) {
+    NSLog(@"Picked label:");
 
-        return;
+    for (id key in [labelPickResult properties]) {
```

Comment:
I've been using vim so I was not aware of that, but will update too if that makes code completion easier.